packages/extract-package-details/src/extractPackageDetails.ts

```diff
@@ -0,0 +1,23 @@
+import path from "path";
+import { GitHubActionsContext } from "@octopusdeploy/shared-action-utils";
+
+export function extractPackageDetails(context: GitHubActionsContext) {
+    try {
+        let dirPath = context.getInput("path");
+        if (dirPath === "") {
+            throw new Error("Path must be provided");
+        }
+        dirPath = path.resolve(dirPath);
+
+        const packageJsonPath = path.join(dirPath, "package.json");
+        const packageJson = require(packageJsonPath);
```

Comment:
I had a few problems requiring in the package json path in the final built version running within an action when working on the other add-changeset action, I had to switch to reading the file contents and then doing `JSON.parse`. Be interesting to see if the same things occur here as I was doing an `import` instead.